docs/src/rules/func-style.md

```diff
@@ -148,6 +153,90 @@ var foo = () => {};
 
 :::
 
+### overrides
+
+#### namedExports
+
+##### expression
+
+Examples of **iincorrect** code for this rule with the `"declaration"` and `{"overrides": { "namedExports": "expression" }}` option:
+
+::: incorrect
+
+```js
+/*eslint func-style: ["error", "declaration", { "overrides": { "namedExports": "expression" } }]*/
+
+function foo() {
+    // ...
+}
+```
```

Comment:
i think in these example we should show exported function
```js
export function foo() {}
```

or both declared and exported declared functions

```js
function foo() {}

export function bar() {}
```

can you update all examples?

Reply:
Sorry, forgetting `export` was a huge mistake. And non-exported use of function declarations is correct with this option. So we just need add missing `export` for examples